src/js/claims-status/containers/AppealInfo.jsx

```diff
@@ -82,17 +103,17 @@ export class AppealInfo extends React.Component {
       // Yes, we have your appeals. No, the one you requested isn't one of them.
       appealContent = <AppealNotFound/>;
     } else if (appealsAvailability === USER_FORBIDDEN_ERROR) {
```

Comment:
It looks like there are only two cases here, can we consolidate these into a smaller block of code?

Reply:
Yeah, I thought about consolidating them all. I guess I'm more assuming we'll have a reason to break out the availability into the different "types" of unavailable in the first place, so this leaves it more open to modification.

On the other hand, if it needs to change later, it's super easy to pull out a specific case, so... :man_shrugging: 

I'll go ahead and consolidate these. It's cleaner. :+1: